alert-engine/src/main/scala/io/phdata/pulse/alertengine/AlertEngineMain.scala

```diff
@@ -44,13 +45,39 @@ object AlertEngineMain extends LazyLogging {
 
     val executorService = Executors.newSingleThreadScheduledExecutor()
 
+    val solrServer = new CloudSolrServer(parsedArgs.zkHost())
+
+    val mailNotificationService =
+      new MailNotificationService(parsedArgs.smtpServer(),
+        parsedArgs.smtpPort(),
+        parsedArgs.smtpUser(),
+        parsedArgs.smtpPassword,
+        parsedArgs.smtp_tls())
+
+    val slackNotificationService = new SlackNotificationService
+
+    val notificationFactory =
+      new NotificationServices(mailNotificationService, slackNotificationService)
+
+    val config = try {
+      AlertEngineConfigParser.getConfig(parsedArgs.conf())
+    } catch {
+      case e: Exception =>
+        logger.error("Error parsing configuration, exiting", e)
+        System.exit(1)
+        throw new RuntimeException(e) // this code won't be reached but is needed for the typechecker
```

Comment:
Also this comment can be removed, and the line can just be `throw e`

Reply:
OK, it's been updated.